packages/search/addon/services/navi-search-provider.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * This service is used to discover all the available search providers.
+ */
+
+import Service from '@ember/service';
+import { getOwner } from '@ember/application';
+import config from 'ember-get-config';
+
+/* global requirejs */
+
+export default class NaviSearchProviderService extends Service {
+  /**
+   * @method getProvider
+   * @param name
+   * @returns {Object} search provider service object
+   */
+  getProvider(name) {
+    return getOwner(this).lookup(`service:navi-search/${name}`);
+  }
+
+  /**
+   * @method all – Discovers all the search provider services
+   * under the folder services/navi-search/
+   * @returns {Array} array of available search provider services
+   */
+  all() {
```

Comment:
Is this method meant to be used outside this service? If not, should this be private?